old-ui/app/components/send/send-contract.js

```diff
@@ -174,16 +213,28 @@ class SendTransactionScreen extends PersistentForm {
 				/> : null}
 			</h3>
 		)
-		const field = (
-			<SendTransactionField
-				key={Math.random()}
-				ind={ind}
-				disabled={!isInput}
-				placeholder={params.type}
-				defaultValue={defaultValue}
-				onChange={e => isInput ? this.handleInputChange(e.target.value, params.type, ind) : null}
-			/>
-		)
+		let field
+		if (params.type === 'bool' && isInput) {
+			field = (
+				<SendTransactionInputSelect
+					key={Math.random()}
```

Comment:
Why is this necesssary?

Reply:
Indeed, `key` prop is not necessary for inputs. Removed, thanks @fvictorio